data/json/itemgroups/Locations_MapExtras/locations_commercial.json

```
@@ -1205,7 +1205,7 @@
       { "item": "mask_dust", "prob": 15 },
       { "item": "manual_mechanics", "prob": 35 },
       { "item": "manual_fabrication", "prob": 20 },
-      { "item": "duct_tape", "prob": 70, "charges": [ 50, 200 ] },
+      { "item": "cardboard_roll", "prob": 70 "ammo-item": "duct_tape", "charges": [ 50, 200 ] },
```

```suggestion
      { "item": "cardboard_roll", "prob": 70, "ammo-item": "duct_tape", "charges": [ 50, 200 ] },
```

JSON loading error otherwise